sql/testdata/system

```diff
@@ -0,0 +1,27 @@
+query T
+SHOW DATABASES
+----
+system
+test
+
+query T
+SHOW TABLES FROM system
+----
+descriptor
+namespace
+
+query ITTB
+EXPLAIN (DEBUG) SELECT * FROM system.namespace
+----
+0 /namespace/primary/0/'system'/id     1    true
+1 /namespace/primary/0/'test'/id       1000 true
+2 /namespace/primary/1/'descriptor'/id 3    true
+3 /namespace/primary/1/'namespace'/id  2    true
+
+query I
+SELECT id FROM system.descriptor
+----
+1
+2
+3
+1000
```

Comment:
`SELECT * FROM system.descriptor` is very ugly due to the encoded protobufs. I've added `SELECT * FROM system.namespace`.